classes/class-log.php

```diff
@@ -194,7 +194,12 @@ public function is_record_excluded( $connector, $context, $action, $user = null,
 				'role'       => ( ! empty( $exclude_rule['author_or_role'] ) && ! is_numeric( $exclude_rule['author_or_role'] ) ) ? $exclude_rule['author_or_role'] : null,
 			);
 
-			$exclude_rules = array_filter( $exclude, 'strlen' );
+			$exclude_rules = array_filter(
+				$exclude,
+				function ( $value ) {
+					return ! is_null( $value ) && '' !== $value;
```

Comment:
I absolutely love your reviews, @tharsheblows! You're totally right about the second check (removed in 9d47e25) - in this scenario it's redundant since we are explicitly setting every empty property to `null` above.

One of the solutions I was considering and trying out involved setting the properties above to empty strings instead of `null` so that the original `strlen`-based solution would still work. I guess I got lost somewhere in between 😅